mm/2s2h/Rando/Logic/Regions.cpp

```diff
@@ -292,6 +297,47 @@ std::unordered_map<RandoRegionId, RandoRegion> Regions = {
             EXIT(ENTRANCE(MOUNTAIN_VILLAGE_WINTER, 3),  ENTRANCE(GORON_GRAVERYARD, 0), true),
         },
     } },
+    { RR_GREATBAY_COAST, RandoRegion{ .sceneId = SCENE_30GYOSON,
+        .checks = {
+            CHECK(RC_GREAT_BAY_COAST_HP,            HAS_ITEM(ITEM_HOOKSHOT) && HAS_ITEM(ITEM_MAGIC_BEANS) && HAS_ITEM(ITEM_OCARINA_OF_TIME) && CAN_BE_HUMAN && (HAS_ITEM(ITEM_SPRING_WATER) || CHECK_QUEST_ITEM(QUEST_SONG_STORMS))),
+            CHECK(RC_GREAT_BAY_COAST_MIKAU,         CAN_BE_HUMAN || CHECK_QUEST_ITEM(QUEST_SONG_HEALING)),
```

Comment:
Should this be `CAN_BE_HUMAN && CHECK_QUEST_ITEM(QUEST_SONG_HEALING)`?

Reply:
Oh yeah, my bad haha, ill go correct that. Thanks for pointing it out!

Reply:
Actually, we should also probably add the Ocarina as a requirement too.